drafts/kin.kinetics-silica-metasomatism.py

```diff
@@ -0,0 +1,258 @@
+# 	-Vm		24.63
+# #	Extrapol	supcrt92
+# #	Ref		HDN+78
+# .setEquation("Brucite = Mg++ + 2*OH-") \
+# .setEquation("Brucite + 2*H+ = Mg++ + 2*H2O(l)") \
+editor.addMineralReaction("Brucite") \
+    .setEquation("Brucite + 2*H+ = Mg++ + 2*H2O(l)") \
+    .addMechanism("logk = -8.24 mol/(m2*s); Ea = 42.0 kJ/mol") \
+    .addMechanism("logk = -4.73 mol/(m2*s); Ea = 59.0 kJ/mol; a[H+] = 0.5") \
+    .setSpecificSurfaceArea(10, "cm2/g")
+
+# Mg3Si4O10(OH)2
+# Talc
+# 	Mg3Si4O10(OH)2 + 4 H2O + 6 H+ = 3 Mg+2 + 4 H4SiO4
+# 	-log_k	21.399
+# 	-delta_h -46.352 kcal
+# 	-Vm 68.34
+# Talc
+# 	Mg3Si4O10(OH)2 + 6 H+ = 3 Mg+2 + 4 H2O + 4 SiO2
+# 	log_k		21.1383
+# 	-delta_H	-148.737	kJ/mol
+# #	deltafH		-1410.92	kcal/mol
+# 	-analytic	1.1164e1 2.4724e-2 1.9810e4 -1.7568e1 -1.8241e6
+# #	Range		0-350
+# 	-Vm		136.25
+# #	Extrapol	supcrt92
+# #	Ref		HDN+78, Wilson+06 match
+editor.addMineralReaction("Talc") \
+    .setEquation("Talc + 6*H+ = 3*Mg++ + 4*SiO2(aq) + 4*H2O(l)") \
+    .addMechanism("logk = -0.21 mol/(m2*s); Ea = 7.4 kJ/mol") \
+    .setSpecificSurfaceArea(10, "cm2/g")
+
+# Mg3Si2O5(OH)4
+# Chrysotile
+# 	Mg3Si2O5(OH)4 + 6 H+ = 2 SiO2 + 3 Mg+2 + 5 H2O
+# 	log_k		31.1254
+# 	-delta_H	-218.041	kJ/mol
+# #	deltafH		-1043.12	kcal/mol
+# 	-analytic	-9.2462e1 -1.1359e-2 1.8312e4 2.9289e1 -6.2342e5
+# #	Range		0-350
+# 	-Vm		108.5
+# #	Extrapol	supcrt92
+# #	Ref		HDN+78
+editor.addMineralReaction("Chrysotile") \
+    .setEquation("Chrysotile + 6*H+ = 2*SiO2(aq) + 3*Mg++ + 5*H2O(l)") \
+    .addMechanism("logk = -12.00 mol/(m2*s); Ea = 73.5 kJ/mol") \
+    .setSpecificSurfaceArea(10, "cm2/g")
+
+reactions = ReactionSystem(editor)
+# -
+
+# Specifying the partition including the kinetic species:
+partition = Partition(system)
+partition.setKineticSpecies(["Forsterite", "Brucite", "Talc", "Chrysotile"])
+
+
+T = 300.0 + 273.15       # temperature (in units of celsius)
+P = 8.58 * 1e5   # pressure (in units of atm)
+problem_ic = EquilibriumProblem(system)
+problem_ic.setTemperature(T)
+problem_ic.setPressure(P)
+problem_ic.add('H2O', 1.0, 'kg')
+problem_ic.add('SiO(aq)', 7.12 * 1e-2, "mol") # 7.12 * 1e-8 mol/cm3 = 7.12 * 1e-2 mol / m3  # MM(SiO2) = 60.08 g/mol
+#problem_ic.add('Forsterite', 10.0, 'mol')
+#problem_ic.add('Mg2SiO4', 10.0, 'mol')
+
+# Calculating the initial chemical equilibrium state of the fluid
+
+# Calculate the equilibrium states for the initial conditions
+state_ic = equilibrate(problem_ic)
+print(state_ic)
+state_ic.output('kineticpath-silica-metasomatism-initial-state.txt')
+# Set the minerals initial values:
+
+# +
+# Clay minerals have the following proportions:
+# Quartz        SiO2            85 %
+# Calcite       CaCO3           6 %
+# Dolomite      CaMg(CO3)2      4 %
+# K-feldspar    K(AlSi3)O8      3 %
+# Kaolinite     Al2Si2O5(OH)4   2 %
+
+# Scale the volumes of the phases in the initial condition
+#state_ic.scalePhaseVolume('Aqueous', 0.37, 'm3') # 37%
+#state_ic.scalePhaseVolume('Forsterite', 0.63, 'm3') # for 63%
```

Comment:
Are these comments (I also see other commented codes in the other files) meant to be here?